docker-based-ror-dev-env/runInDocker.sh

```diff
@@ -30,6 +30,7 @@ fi
 
 docker run --rm $DIND_OPTIONS $DOCKER_RUN_OPTIONS \
   -e ROR_ACTIVATION_KEY="$ROR_ACTIVATION_KEY" \
+  -e I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING=yes \
```

Comment:
let's just hardcode it in the Dockerfile in this line:
```
$INVOKE_ROR_TOOLS patch && \
```

Reply:
Done ✔️